test/e2e/fixture-builder.js

```diff
@@ -0,0 +1,659 @@
+class Fixture {
```

Comment:
Nit: Not sure that making this a `class` is helpful. We aren't really using any class features, aside from that the default fixture state is "protected" against accidental modification.

We can more simply protect it by making this a function that returns this object. Or by making it a constant and running `deep-freeze-strict` on it.